Makefile

```diff
@@ -79,6 +82,15 @@ all-build: $(addprefix build-, $(CLI_PLATFORMS))
 
 #all-push: $(addprefix push-, $(CONTAINER_PLATFORMS))
 
+local: build-dirs
+	GOOS=$(GOOS) \
```

Comment:
I _think_ we can omit specifying all the variables that are unmodified and `make` should pass them through when executing `build.sh`. That would leave `OUTPUT_DIR` as the only one we need. Try that and see if it works?

Reply:
nope :(

Reply:
Aha, we could `export` whatever variables we want to be passed down to *all* shell invocations from all targets, if desired.

Reply:
Eh, maybe in a separate PR :)